src/website/views/components/popover/_examples.pug

```diff
@@ -220,6 +220,72 @@ p.-text
       <button id='popover-1' class="chi-button" data-popover-content='<header class="chi-popover__header"><h2 class="chi-popover__title">Popover title</h2></header><div class="chi-popover__content"><p class="chi-popover__text">Popover content</p></div>'>Popover</button>
       <script>chi.popover(document.getElementById('popover-1'));</script>
 
+h3 Open on hover
```

Comment:
@janibolkvadze examples are not working on IE11, maybe it's related to JS arrow function.